### PR TITLE
hack/e2e-node-test should not assume ginkgo installed

### PR DESCRIPTION
**Pull request by eparis**
It should use the version we build.

### CODE REVIEW

**Comment by eparis** [comment]
@derekwaynecarr @pwittrock

**Comment by eparis** [comment]
It doesn't work mind you, but at least it doesn't try to use my local GOPATH and instead uses the stuff we have in Godeps, including the ginkgo we build out of godeps

**Comment by derekwaynecarr** [comment]
@eparis 

hack/e2e-node-test.sh
hack/e2e-node-test.sh: line 24: : command not found
!!! Error in hack/e2e-node-test.sh:24
  '"${ginkgo}" "${KUBE_ROOT}/test/e2e_node/" -- --alsologtostderr --v 2 --node-name $(hostname) --build-services=true --start-services=true --stop-services=true' exited with status 127
Call stack:
  1: hack/e2e-node-test.sh:24 main(...)
Exiting with status 1
Makefile:109: recipe for target 'test_e2e_node' failed
make: *** [test_e2e_node] Error 1

**Comment by eparis** [comment]
PR updated to notice ginkgo wasn't build and to tell you how to build it.

**Comment by derekwaynecarr** [comment]
LGTM, was able to start running tests...

File: hack/e2e-node-test.sh [hunk 1]
```diff
@@ -14,8 +14,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+kube::golang::setup_env
+
+ginkgo=$(kube::util::find-binary "ginkgo")
```
Comment (lavalamp) [inline hack/e2e-node-test.sh]: local? or does that only work in functions. I forget.

Comment (eparis) [inline hack/e2e-node-test.sh]: functions

**Pull request merged**